components/profileImageComponents/ProfileImageAndName.tsx

```diff
@@ -0,0 +1,136 @@
+      <ProfileImage imageURL="@assets/icon.png" style={imageStyle} />
+      <View>
+        <View style={{ flexDirection: "row" }}>
+          <Headline>{username}</Headline>
+          {displayFriend &&
+            (!isFriend
+              ? (
+                !requestSent
+                  ? (
+                    <View style={[{ flexDirection: "row", alignItems: "center" }]}>
+                      <Ionicon
+                        style={{ marginHorizontal: 8 }}
+                        size={6}
+                        color={AppStyles.colorOpacity35}
+                        name="ellipse"
+                      />
+                      <Headline
+                        onPress={() => setRequestSent(true)}
+                        style={{ color: eventColor }}
+                      >
+                    Add Friend
+                      </Headline>
+                    </View>
+                  )
+                  : (
+                    <View style={[{ flexDirection: "row", alignItems: "center" }]}>
+                      <Ionicons
+                        style={{ marginHorizontal: 8 }}
+                        color={AppStyles.colorOpacity35}
+                        size={6}
+                        name="ellipse"
+                      />
+                      <Headline
+                        style={{ opacity: 0.35 }}
+                        onPress={() => setRequestSent(true)}
+                      >
+                    Request Sent
+                      </Headline>
+                      <Toast
+                        visible={requestSent}
+                        opacity={1}
+                        position={Toast.positions.BOTTOM - offset}
+                        shadow={false}
+                        animation={true}
+                        hideOnPress={true}
+                        containerStyle={styles.toastStyle}
+                      >
+                        <View style={{ flexDirection: "row" }}>
+                          <View style={{ marginRight: 16 }}>
+                            <Ionicon color="white" name="close" />
+                          </View>
+                          <BodyText
+                            style={{ color: "white", textAlignVertical: "center" }}
+                          >
+                            {"Friend request sent"}
+                          </BodyText>
+                        </View>
+                      </Toast>
+                    </View>
+                  )
+              )
+              : (
+                <View>
+                  <Ionicons
+                    style={{ marginHorizontal: 8 }}
+                    color={AppStyles.colorOpacity35}
+                    size={6}
+                    name="ellipse"
+                  />
+                  <Headline
+                    style={{ opacity: 0.35 }}
+                    onPress={() => setRequestSent(true)}
+                  >
+                  Friend
```

Comment:
```suggestion
                  Friends
```
The plural sounds more appealing when viewing the UI

Reply:
Yeah, you're right.